packages/logo-react/example/index.tsx

```diff
@@ -1,32 +1,50 @@
 import React, { useState } from "react";
 import ReactDOM from "react-dom";
-import { LogoSymbol, LogoMain, LogoAnimated } from "../src";
+import {
+    LogoSymbol,
+    LogoMain,
+    LogoAnimated,
+    LogoStampTechnology,
```

Comment:
What do you think about `<LogoStamp type="technology" />` etc.? There might be more of these later, and we'll end up with a lot of components..